PAPER/FIT_Paper_Ex1/Part_1.tex

```diff
@@ -1,52 +1,56 @@
-Since the 1990's, many different nuclear fuel cycle simulation codes have been
+Since the 1990's, many different nuclear fuel cycle simulators have been
 developed by several institutions.
 
-A fuel cycle code aims to models an entire nuclear fleet including main
-facilities, like nuclear reactors, fuel fabrication plants, reprocessing plants,
-cooling pool and waste disposal. Those tools help to identify drivers and
-interactions between parameters in fuel cycle physic. For that, they often
-implement physic models for different key points of the cycle like fuel
-fabrication or burn up depletion calculations. 
+A fuel cycle simulator aims to models an entire nuclear fleet including main
+facilities, such as nuclear reactors, fuel fabrication plants, reprocessing
+plants, cooling pool and/or waste disposal. Those tools help to identify drivers
+and interactions between parameters in fuel cycle. They implement physic models
+for different key points of the cycle such as fuel fabrication or burn up
+depletions, with various level of complexicity. 
 
-Fuel cycle simulation codes are used worldwide for different purposes. Fuel
-cycle tools can be used to study current nuclear fleet for industrial operation
-optimization. Those tools can also be used to assess the future of nuclear
-energy and provide then informations for political decision process. Finally,
-those tools may be used for Research and Development training as they give an
-access to key point data related to the fuel cycle. 
+Fuel cycle simulator are used worldwide for wide range of applications:
+optimisation of the industrial operation of a existing nuclear fleet, assessing
+the future of nuclear energy and providing valuable informations required for
+the political decision, developement of tools dedicated to non-proliferation
+treaties verification. Moreover, those tools are used for Research and
+Development training as access point to key datas related to the fuel cycle. 
 
-The different institutions that use fuel cycle simulations pursue different
-goals. Consequently, their computational tools vary in term of complexity. The
-level of complexity of each software has to be coherent in regards to the
-different studies it is used for. To improve confidence in the results,
-institutions are tempted to increase the complexity of their software even if
-this complexity might not be necessary.
+The different institutions that use (and develop) fuel cycle simulators pursue
+different goals. Consequently, their computational needs vary in term of
+complexity and the level of complexity of each software pieces has to be in
+adequation with regards to their different simulations requirements. To improve
```

Comment:
Replace "simulation requirement" with a term that say "what do you want to do with your soft. What is the study you want to realize." This was said before with "different studies it is used for". Maybe we should find better.

Reply:
simulations needs ?

Reply:
simulation's goals ?